ibm/service/power/resource_ibm_pi_host_group.go

```diff
@@ -0,0 +1,323 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/go-sdk-core/v5/core"
+)
+
+func ResourceIBMPIHostGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPIHostGroupCreate,
+		ReadContext:   resourceIBMPIHostGroupRead,
+		DeleteContext: resourceIBMPIHostGroupDelete,
+		UpdateContext: resourceIBMPIHostGroupUpdate,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			// Arguments
+			Arg_CloudInstanceID: {
+				Description: "PI cloud instance ID",
+				ForceNew:    true,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+			Arg_Hosts: {
+				Description: "List of hosts to add to the group.",
+				ForceNew:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						Attr_DisplayName: {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Name of the host chosen by the user.",
+						},
+						Attr_SysType: {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "System type.",
+						},
+					},
+				},
+				Required: true,
+				Type:     schema.TypeList,
+			},
+			Arg_Name: {
+				Description: "Name of the host group to create.",
+				ForceNew:    true,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+			Arg_remove: {
+				Description: "A workspace ID to stop sharing the host group with.",
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			Arg_Secondaries: {
+				Description: "List of workspace names to share the host group with.",
```

Comment:
I think this description has been updated say workspace IDs in the model. The description looks like is correct in the markdown